cmd.md

```diff
@@ -142,11 +174,20 @@ OR
 
 - Attributes include 'archived', 'hidden', 'system', etc.
 - `attrib /?` for overview.
-- Go to dir and type `attrib` to view the attributes of the files in it.
+- Go to directory and type `attrib` to view the attributes of the files in it.
 - Adding attributes, eg: `attrib -h +r file_name.txt`.
 
-## Background and text colour
+## Background and text color
+
+- `color bt` (b = background color, t = text color)
+- For color legend, type `color /?`.
+- Type `color` to reset all colors to default.
+
+## Chaining commands
 
-- `color bt` (b = background colour, t = text colour)
-- For colour legend, type `color /?`.
-- Type `color` to reset all colours to default.
+- You can chain multiple commands using `&`, `&&` and `||`.
+- `command_1 & command_2` will run `command_1` and then run `command_2`.
+- `command_1 && command_2` will run `command_1` and then run `command_2` only if `command_1 finished successfully.
+- `command_1 && command_2` will run `command_1` and then run `command_2` only if `command_1 **did not** finished successfully.
```

Comment:
Check point number 3 and 4 of `chaining commands`, there seems to be some grammatical error and formatting errors.

Reply:
Done.